eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration.php

```diff
@@ -568,4 +569,44 @@ function ($v) {
             ->end()
         ;
     }
+
+    /**
+     * Define Url Alias Slug converter Semantic Configuration.
+     *
+     * The configuration is available at:
+     * <code>
+     * ezpublish:
+     *     url_alias:
+     *         transformation: name_of_transformation_group_to_use
+     *         separator:  name_of_separator_to_use
+     *         transformationGroups:
+     *             transformation_group_name: name of existing or new transformation group
+     *                 commands : [] array of commands which will be added to group
+     *                 cleanupMethod: name_of_cleanup_method
+     * </code>
+     * @param \Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition $rootNode
+     *
+     * @return null|\Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition|\Symfony\Component\Config\Definition\Builder\NodeBuilder|\Symfony\Component\Config\Definition\Builder\NodeDefinition|\Symfony\Component\Config\Definition\Builder\NodeParentInterface|\Symfony\Component\Config\Definition\Builder\ParentNodeDefinitionInterface|\Symfony\Component\Config\Definition\Builder\VariableNodeDefinition
+     */
+    private function addUrlAliasSection(ArrayNodeDefinition $rootNode)
+    {
+        return $rootNode
+            ->children()
+                ->arrayNode('url_alias')
+                    ->children()
+                        ->scalarNode('transformation')->end()
+                        ->scalarNode('separator')->end()
```

Comment:
Do we have some sort of well defined list of possible values for `transformation` and `separator`? If so, then value should be validated here.